### PR TITLE
Pydantic v2 support

### PR DESCRIPTION
**Pull request by 50Bytes-dev**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -17,11 +17,11 @@ keywords = ["OpenAPI", "Generator", "Python", "async"]
 Changelog = "https://github.com/MarcoMuellner/openapi-python-generator/releases"
 
 [tool.poetry.dependencies]
-python = "^3.7"
+python = ">=3.11,<3.12"
```
Comment (MarcoMuellner) [inline pyproject.toml:20]: ^3.9 should be fine here

**Review by MarcoMuellner — CHANGES_REQUESTED** [review]
Hey, thank you for the contribution! I like this! Can you make sure that the test workflow still runs and that the tests are adapted accordingly? 

I also think the python version doesn't necessarily need to be locked to 3.11.

**Comment by 305jthompson** [comment]
Any update on this? I have a project that uses Pydantic v2 and I would love to use this in my project. Thanks!

**Comment by MarcoMuellner** [comment]
Implemented by #76

**Pull request closed without merging**